library.js

```diff
@@ -0,0 +1,90 @@
+// declare empty array for library
+let myLibrary = [];
+let i = 0;
+//object constructor
+function Book(Title, Author, Pages, Read) {
+  this.Title = Title;
+  this.Author = Author;
+  this.Pages = Pages;
+  this.Read = Read;
+}
+// function to add a new book to library
+function addBookToLibrary(Title, Author, Pages, Read) {
+  let book = new Book(Title, Author, Pages, Read);
+  myLibrary.push(book);
+  displayBooksOnPage();
+}
+//for displaying the books in library
+function displayBooksOnPage() {
+  const books = document.querySelector(".books");
+  // create card
+  const card = document.createElement("div");
+  card.classList.add("card");
+  books.appendChild(card);
+  for (key in myLibrary[i]) {
```

Comment:
The `i` variable is used as a global variable, which can cause problems in a larger codebase. You should consider making it a local variable within the function instead.

Reply:
Don't use global variable `myLibrary` like this, pass it as an argument in the function `displayBooksOnPage`

Reply:
Ideally a function should not access or update the global function directly.